### ISSUE TITLE
SegmentedButton keyboard focus traversal not working

### ISSUE THREAD

**Issue by JonathanGiles** [controlsfx/controlsfx#87]
**[Original report](https://bitbucket.org/controlsfx/controlsfx/issue/87) by Paul Furbacher (Bitbucket: [pfurbacher](https://bitbucket.org/pfurbacher), GitHub: [pfurbacher](https://github.com/pfurbacher)).**

----------------------------------------

Because the main constructor of SegmentedButton invokes

```
#!java
    setFocusTraversal(true);
```

keyboard focus traversal in the HelloSegmentedButton.java demo app does not work.

You really have to set focus traversal for the segmented button "container" to false:

```
#!java
    public SegmentedButton(ObservableList<ToggleButton> buttons) {
        getStyleClass().add("segmented-button");
        this.buttons = buttons == null ? FXCollections.<ToggleButton>observableArrayList() : buttons;
        setFocusTraversable(**false**);
    }
```

Keyboard traversal is **really** important.
